helm-chart/values.yaml

```diff
@@ -95,9 +95,14 @@ config:
   # Use config.hosted.HEDERA_NETWORK when running against a hosted Hedera network.  Valid options are `previewnet`, `testnet`, and `mainnet`  
   # hosted:
   #   HEDERA_NETWORK: ""
-secret:
   OPERATOR_ID_MAIN: ""
   OPERATOR_KEY_MAIN: ""
   OPERATOR_ID_ETH_SENDRAWTRANSACTION: ""
-  OPERATOR_KEY_ETH_SENDRAWTRANSACTION: ""  
+  OPERATOR_KEY_ETH_SENDRAWTRANSACTION: ""
+
+# Enable rolling_restarts if SDK calls fail this is usually due to stale connections that get cycled on restart
```

Comment:
Can we change the replicaCount to 2 with this PR?
Or would k8s handle this by spinning one pod up first and moving traffic there before dropping the current pod?